src/utils/ExtractTestResults.ts

```diff
@@ -22,10 +22,10 @@ import { ValidationUtil } from './ValidationUtil';
 /**
  * This is used to extract the relevant fields from the test record that is
  * required to be sent to MC in order to  clear prohibitions
- * @param record
+ * @param record : DynamoDBRecord
```

Comment:
```suggestion
 * @param {DynamoDBRecord} record
```
Far as I know, this is how you add the type